code/game/atoms.dm

```diff
@@ -621,3 +621,17 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 
 /atom/proc/return_temperature()
 	return
+
+// Default tool behaviors proc
+
+/atom/proc/crowbar_behavior(mob/user)

```

Comment:
Maybe use `X_act` to be in line with `emag_act` that serves the same purpose?

Reply:
yea, idk yet, my first thought was indeed `x_act` but i'm not really a fan of that name